### PR TITLE
[ZT] cloudflared downloads section

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/1.1.1.1/encryption/dns-over-https/dns-over-https-client.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ There are several DoH clients you can use to connect to 1.1.1.1.
 
 ## cloudflared
 
-1. [Download and install the `cloudflared` daemon](/cloudflare-one/connections/connect-apps/install-and-setup/installation/).
```
Comment (abelinkinbio) [inline content/1.1.1.1/encryption/dns-over-https/dns-over-https-client.md:12]: Is this the right time to change connect-apps to connect-network?

File: content/cloudflare-one/connections/connect-apps/downloads/update-cloudflared.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+If you installed `cloudflared` from GitHub binaries or from source, run the following command:
+
+```sh
+$ cloudflared update
+```
+
+If you installed `cloudflared` with a package manager, you must update it using the same package manager. On Linux, you can check if `cloudflared` is owned by a package manager by running `ls -la /usr/local/etc/cloudflared/` and looking for `.installedFromPackageManager` in the output.
+
+## Update with Cloudflare Load Balancer
```
Comment (abelinkinbio) [inline content/cloudflare-one/connections/connect-apps/downloads/update-cloudflared.md]: We should specify that LB/replicas are zero downtime upgrade paths. If you don't have this requirement the options above are more straightforward paths.

Comment (ranbel) [reply content/cloudflare-one/connections/connect-apps/downloads/update-cloudflared.md]: We call this out at the very top of the page (line 9).

Comment (abelinkinbio) [reply content/cloudflare-one/connections/connect-apps/downloads/update-cloudflared.md]: Good deal.

File: content/1.1.1.1/encryption/dns-over-https/dns-over-https-client.md (hunk 1 above)
Comment (ranbel) [reply content/1.1.1.1/encryption/dns-over-https/dns-over-https-client.md:12]: Can it wait until we tackle subnet routing?

Comment (abelinkinbio) [reply content/1.1.1.1/encryption/dns-over-https/dns-over-https-client.md:12]: Should it? That'll already be a large PR. Any reason why we couldn't include it in this PR?

Comment (ranbel) [reply content/1.1.1.1/encryption/dns-over-https/dns-over-https-client.md:12]: Is it just changing the URL for Tunnel docs?

Comment (abelinkinbio) [reply content/1.1.1.1/encryption/dns-over-https/dns-over-https-client.md:12]: Yea exactly

Comment (ranbel) [reply content/1.1.1.1/encryption/dns-over-https/dns-over-https-client.md:12]: Ok I updated all of the links.

**Review by abelinkinbio — APPROVED** [review]

**Pull request merged**